components/image/picker/src/index.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+import ImagePlaceholder from '@schibstedspain/sui-image-placeholder'
+
+class ImagePicker extends Component {
+  constructor (props) {
+    super()
+
+    const { images } = props
+    let filteredImages = []
+    let selected = -1
+
+    if (images.length > 0) {
+      filteredImages = images.map(image => {
+        if (typeof image.src !== 'undefined' && typeof image.thumb !== 'undefined') { return image }
+        return false
+      }).filter(image => image !== false)
+
+      if (filteredImages.length > 0) {
+        selected = filteredImages.findIndex(image => image.selected === true)
```

Comment:
You don't need the equality as I supposed it's either true or false anyways
```
filteredImages.findIndex({selected}=> selected)
````

Reply:
No. It can even not exist. But I'm taking some idea from your line.

Reply:
To work with undefined:
```
filteredImages.findIndex({selected}=> !!selected)
```